src/components/Layout/Header/NavBar/ChainMenu.tsx

```diff
@@ -116,13 +61,13 @@ export const ChainMenu = memo((props: ChainMenuProps) => {
           rpcUrls: [...maybeGnosisOfficialRpcUrl, requestedChainRpcUrl],
           blockExplorerUrls: [requestedChainFeeAsset.explorer],
         })
-        setEthNetwork(requestedEthNetwork)
+        setEthNetwork(requestedChainId)
```

Comment:
This is not an ETH network (hex in JSON-RPC land but actual int effectively) anymore but a CAIP

Reply:
correct you are a938f56d3beb4fbbaa8ad56aa222bbe78eece622